database/migrations/2017_09_29_161315_remove-subscriptions-fk.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
```

Comment:
Do not remove the fk of the subscriptions to payments table! Make it nullable instead.